inputs/orchestrator_inner:2.json

```diff
@@ -0,0 +1,55 @@
+{
+  "client_version": "{{VERSION}}",
+  "prebuild_plugins": [
+    {
+      "name": "add_filesystem",
+      "args": {
+        "hub": "{{KOJI_HUB}}",
+        "repos": []
+      }
+    },
```

Comment:
I think it can be removed from the orchestrator build now. That's [where we want to get to](http://osbs.readthedocs.io/en/latest/multiarch.html#anatomy-of-an-orchestrator-build) anyway.